src/main/java/TaskListener.java

```diff
@@ -0,0 +1,191 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TaskListener {
+    static final String DIVIDER = "--------------------------------------------------------------------";
+    private TaskList taskList;
+
+    public TaskListener(int expectedTasksCount) {
+        this.taskList = new TaskList(expectedTasksCount);
+    }
+
+    private static void printLines(String... lines) {
+        System.out.println(DIVIDER);
+        for (String line : lines) {
+            System.out.println(line);
+        }
+        System.out.println(DIVIDER + System.lineSeparator());
+    }
+
+    private void printAddedTask(Task task) {
+        String firstSentence = "Got it. I've added this task:";
+        String tasksWord = taskList.getTasksCount() == 1 ? " task" : " tasks";
+        String secondSentence = "Now you have " + taskList.getTasksCount() + tasksWord + " in the list.";
+        printLines(firstSentence + System.lineSeparator() + task + System.lineSeparator() + secondSentence);
```

Comment:
The `firstSentence` and `secondSentence` variables are constants and should be capitalised. Either that, or simply concatenating them into the `printLines()` function should be fine, as this would be more consistent with the other string literals you left in some lines; converting **all** literals into constants is quite an unnecessary task imo. Not sure if the single line if-else statement is allowed, but I'd rather use normal if-else statement for consistency sake.